static/js/telescope_availability_app.vue

```diff
@@ -0,0 +1,31 @@
+<template>
```

Comment:
I was following the example of request_details and compose, which each have a .vue and a .js.